### PR TITLE
129 bytebase guideline

### PR DESCRIPTION
**Pull request by Francois-Werbrouck**
Update of the database ADRs and added the bytebase guideline.

Ref #129

Helps https://github.com/ai-cfia/ailab-datastore/issues/135

### CODE REVIEW

File: adr/006-review-database-change-management.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+If we need to revert changes, we can simply point towards the older version of
+the schema thanks to the semantic versionning. We are currently thinking about
+using a tool to automate this process.
+
+Our decision to adopt this approach was influenced by a combination of multiple
+solutions considered, including : Manual SQL Scripts, Database Migration Tools,
+Blue-Green Deployment, and the concept of Database Views and Abstraction Layers.
+
+## Improvement
```
Comment (k-allagbe) [inline adr/006-review-database-change-management.md:68]: To keep in the same spirit of the ADRs, we probably should compare other tools similar to bytebase if there are any.

Comment (Francois-Werbrouck) [reply adr/006-review-database-change-management.md:143]: Done! 👍🏻

**Comment by Francois-Werbrouck** [comment]
There is a new ADR (006-review-database-management) which updates the old ADR-006 to better describe the persisting problems and the updates on our previous decision
 
I've added a  📦 `Bytebase/` including:
- Wins
  - Documentation of all the advantages of using Bytebase including all the features behing a pay wall
- usage-guideline
  - Documentation for the devs concerning the core features and how to use/navigate through them.
 
Please go ahead and review all the documentation. I will make sure it is well built for someone outside the scope of our team (especially the Wins.md which could be quite confusing). I want to make sure the decision is well documented if we still continue towards this avenue. 
 
I also want to make it accessible for Upper-Management, so non-dev feedback is essential (@nmeraihi) if something is not clear enough, let me know and I will go ahead to make it more accessible/clearer)

File: adr/006-review-database-change-management.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+* Offers customizable workflow to verify the changes (GitOps)
+
+Cons:
+
+* Some features are hiddent behind a pay wall (most of those features dont
+    seem to be necessary atm)
+
+* Still in an early development phase
+
+#### Liquipedia
```
[the remark this answers is not in the document]
Comment (Francois-Werbrouck) [reply adr/006-review-database-change-management.md:105]: its liquibase. My brain got mixed up with Liquipedia which is a Esport tracking website lmao. I will fix the typo

File: adr/006-review-database-change-management.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+* Offers a GUI (usefull for unexperienced devs)
+
+* Open source
+
+* Offers a Cl/CD workflow (not a GitOps one)
+
+Cons:
+
+* Does not support Azure PostgreSQL instance
```
[the remark this answers is not in the document]
Comment (Francois-Werbrouck) [reply adr/006-review-database-change-management.md:137]: image
https://www.red-gate.com/products/flyway/community/

File: adr/006-review-database-change-management.md [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+* Integrates with all common CI and release tools, including GitHub and Azure
+  DevOps.
+
+Cons:
+
+* Most features are hidden behind a paywall
+
+* Offers a Git client under paywall
+
+* Not an
```
Comment (k-allagbe) [inline adr/006-review-database-change-management.md:144]: Incomplete?

Comment (Francois-Werbrouck) [reply adr/006-review-database-change-management.md]: This was supposed to be deleted. Probably there after a Ctrl-Z when my cat walked on my keyboard... fixed

**Review by k-allagbe — APPROVED** [review]

**Pull request merged**